server/calendar/calendar-router.js

```diff
@@ -0,0 +1,77 @@
+var calendarController = require('./calendar-controller.js');
+var express = require('express');
+var calendarRouter = express.Router();
+var User = require('./User');
+
+
+calendarRouter.get('/', calendarController.signin);
+
+calendarRouter.get('/callback', calendarController.callback);
+
+calendarRouter.post('/checkloggedin', calendarController.isAuthenticated);
+
+calendarRouter.get('/logout', calendarController.logout);
+
+//Temporary Failure route if passport authentication fail
+calendarRouter.get('/failure', function() {
+  console.log('Something went wrong in authenticating with google');
+});
+
+//Temporary route to get all users calendar meta-information
+calendarRouter.get('/allCalendars', function (req, res) {
```

Comment:
Oh! I understand now - these methods will be used elsewhere in the app (for example when we add an event to :organization/gigs/, and the routes that correspond with these methods will not exist. right now, the routes exist just for testing that they function correctly.

we spoke briefly about abstracting out a calendar module - we should keep that in mind going forward.